contacts/models.py

```diff
@@ -0,0 +1,34 @@
+from django.contrib.auth.base_user import AbstractBaseUser
+from django.db import models
+from django.contrib.auth.models import User
```

Comment:
Use the `get_user_model()` function. It has the benefit of working in case the user model has been reconfigured..

Reply:
i basically don't want all fields in this case thats why i removed my 'User' model. why i need get_user_model in a model like this

`class Article(models.Model):
    author = models.ForeignKey(
        settings.AUTH_USER_MODEL,
        on_delete=models.CASCADE,
    )`

Reply:
Okay, cool. That works as well.